mainnet/14-wrong-permissions/script.js

```diff
@@ -0,0 +1,34 @@
+const {
+  voting,
+  script: { acl, kernel, vaultAddress, appId, appBase, entity, role, manager },
+} = require('./metadata')
+const {
+  encodeMaliciousInstallation,
+  encodeCallsScript,
+} = require('../../lib/encoder')
+const {
+  buildNonceForAddress,
+  calculateNewProxyAddress,
+} = require('../../lib/web3-utils')
+
+module.exports = async () => {
+  // This variable is needed as if we leave the index param in the `buildNonceForAddress`
+  // function as 0, it will calculate the ACL address. For determining what number to assign,
+  // we need to count the number of apps installed, and then add 1.
+  const appIndex = 10.
```

Comment:
Let's move this variable to one of the config parameters in the `script` object of the `metadata` file

Reply:
Done in https://github.com/aragon/court-precedence-campaign/pull/3/commits/6969a24a5b1c132512ea0c05274b86d059795eb3